.github/workflows/ci.yml

```diff
@@ -156,7 +247,7 @@ jobs:
       bucket-name-main: 'pie-storybook'
     secrets: inherit
 
-  browser-tests:
+  browser-tests-docs:
```

Comment:
Is this job missing an `if` check for `output.pie-docs-change == 'true'` perhaps?

Reply:
Oh! I noticed this won't run as it depends on the deploy-docs step, ignore my previous comment :D